charts/newrelic-infrastructure-v3/templates/_helpers.tpl

```diff
@@ -214,10 +222,25 @@ Returns Custom Attributes even if formatted as a json string
 {{- merge (include "newrelic.customAttributesWithoutClusterName" . | fromJson) (dict "clusterName" (include "newrelic.cluster" .)) | toJson }}
 {{- end -}}
 
+{{/*
+Returns integration configmap data,
+if the new one is defined we ignore the old one
+*/}}
+{{- define "newrelic.integrations" -}}
+{{- if .Values.integrations_config -}}
+{{- range .Values.integrations_config }}
+{{ .name -}}: |-
+  {{- toYaml .data | nindent 2 }}
+{{- end -}}
+{{- end -}}
+{{- end -}}
```

Comment:
Do we need a helper for this? We only render it in one place.

Reply:
yes, I added that since we need the same code it in two different places

Reply:
actually they old and new config were mixed up